Source/WebCore/page/MemoryRelease.cpp

```diff
@@ -126,7 +126,7 @@ static void releaseCriticalMemory(Synchronous synchronous, MaintainBackForwardCa
         GCController::singleton().deleteAllCode(JSC::DeleteAllCodeIfNotCollecting);
         GCController::singleton().garbageCollectNow();
     } else {
-#if PLATFORM(IOS_FAMILY)
+#if PLATFORM(IOS_FAMILY) || PLATFORM(WPE)
```

Comment:
I could swear I had added a comment about this, but I guess I didn't submit it in the end, sorry. This shouldn't be needed either for what we're trying to do, as the synchronous case not handled there.

Reply:
Removed as it does not seem to influence based on latest testing (with the other changes)